controllers/notification/internal/cloudclient/client.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright 2023 yxxchange@163.com.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cloudclient
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+
+	ntf "github.com/labring/sealos/controllers/common/notification/api/v1"
+	"github.com/labring/sealos/pkg/utils/logger"
+)
+
+type CloudText struct {
+	ID         string   `json:"_id"`
+	Type       string   `json:"Type"`
+	Title      string   `json:"Title"`
+	Message    string   `json:"Message"`
+	Timestamp  int64    `json:"Timestamp"`
+	Importance ntf.Type `json:"Importance"`
```

Comment:
This part is duplicated with NotificationSpec, directly refer to NotificationSpec:
https://github.com/labring/sealos/pull/3103/files#diff-ddd510a0a410bcd5e1706c9647092640938c0f1ffc0ff39f794bb8523544e0d5R31